test/javascripts/acceptance/composer-test.js

```diff
@@ -22,6 +22,13 @@ acceptance("Discourse Post Voting - composer", function (needs) {
         createAsPostVotingSetInRequest = true;
       }
 
+      if (
+        parsePostData(request.requestBody).only_post_voting_in_this_category ===
+        "true"
+      ) {
+        createAsPostVotingSetInRequest = true;
+      }
+
```

Comment:
You could combine this with the condition above

```suggestion
const parsedData = parsePostData(request.requestBody);
if (
  parsedData.create_as_post_voting === "true" ||
  parsedData.only_post_voting_in_this_category === "true"
) {
  createAsPostVotingSetInRequest = true;
}
```